src/pages/hacks/re-entrancy/ReEntrancy.sol

```diff
@@ -58,16 +58,16 @@ contract Attack {
         etherStore = EtherStore(_etherStoreAddress);
     }
 
-    // Fallback is called when EtherStore sends Ether to this contract.
-    fallback() external payable {
-        if (address(etherStore).balance >= 1 ether) {
+    // Receive is called when EtherStore sends Ether to this contract
+    receive() external payable {
```

Comment:
Please revert this to `fallback`. Fallback is more useful since the attack doesn't care if the caller (`EtherStore`) sends  arbitrary data or not